api/lib/app.js

```diff
@@ -1,19 +1,21 @@
 import * as Sentry from '@sentry/node'
 import Fastify from 'fastify'
+import pg from '@fastify/postgres'
 
 /** @typedef {import('@filecoin-station/deal-observer-db').PgPool} PgPool */
 
 /**
  * @param {object} args
- * @param {PgPool} args.pgPool
+ * @param {string} args.DATABASE_URL
  * @param {Fastify.FastifyLoggerOptions} args.logger
  * @returns
  */
 export const createApp = ({
-  pgPool,
+  DATABASE_URL,
   logger
 }) => {
   const app = Fastify({ logger })
+  app.register(pg, { connectionString: DATABASE_URL })
   Sentry.setupFastifyErrorHandler(app)
```

Comment:
```suggestion
  Sentry.setupFastifyErrorHandler(app)
  app.register(pg, { connectionString: DATABASE_URL })
```

Sentry should always come first